packages/react/src/toaster/Toaster.tsx

```diff
@@ -0,0 +1,32 @@
+import {
+  ToastProvider as RadixToastProvider,
+  ToastViewport as RadixToastViewport,
+} from "@radix-ui/react-toast";
+import { type ComponentPropsWithRef, forwardRef } from "react";
+
+import { Flex } from "../flex";
+import { Toast } from "../toast/Toast";
+import * as styles from "./Toaster.css";
+import { useToast } from "./use-toast";
+
+type ToasterProps = ComponentPropsWithRef<typeof RadixToastProvider> &
+  styles.ToastPositionVariants;
+
+export const Toaster = forwardRef<HTMLDivElement, ToasterProps>(
+  ({ position = "bottom-right", ...props }, ref) => {
+    const { toasts } = useToast();
+    return (
+      <RadixToastProvider>
+        <Flex asChild ref={ref} {...styles.viewPort({ position })} {...props}>
+          <RadixToastViewport>
+            {toasts.map(({ id, ...props }) => (
+              <Toast key={id} {...props} />
+            ))}
+          </RadixToastViewport>
+        </Flex>
```

Comment:
toasts shouldn't be inside the view port children
```suggestion
        {toasts.map(({ id, ...props }) => (
          <Toast key={id} {...props} />
        ))}
        
        <Flex asChild ref={ref} {...styles.viewPort({ position })} {...props}>
          <RadixToastViewport />
        </Flex>
```